pkg/instancetype/instancetype.go

```diff
@@ -527,6 +528,15 @@ func applyCpu(field *k8sfield.Path, instancetypeSpec *instancetypev1alpha2.Virtu
 		return Conflicts{field.Child("domain", "cpu")}
 	}
 
+	hasCPUGuest := false
+	if condition := instancetypeSpec.CPU.Guest; condition != 0 {
```

Comment:
Please move these conflict conditions to separate function (something like `resolveCPUConflicts`).